crypto/keyring/keyring.go

```diff
@@ -180,10 +179,11 @@ func New(
 		db  keyring.Keyring
 		err error
 	)
+	cdc := codec.NewAminoCodec(codec.NewLegacyAmino()) //?
```

Comment:
Is it correct way to pass cdc?

Reply:
No, we shouldn't create a new `cdc`. It should be a new function argument.

Reply:
`cdc` comes from the App.